.github/workflows/aks.yaml

```diff
@@ -29,6 +29,7 @@ env:
   name: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
   location: westeurope
   cost_reduction: --node-vm-size Standard_B2s --node-osdisk-size 30
+  cilium_version: v1.11.4
```

Comment:
Do we actually need these since we are already setting the right value in `values.yaml`? I personally would say we should we do one or the other, but probably not both.